abrt.spec.in

```diff
@@ -298,6 +298,7 @@ uncaught exception in python programs.
 %package -n python2-abrt-container-addon
 Summary: %{name}'s container addon for catching Python 2 exceptions
 Conflicts: python2-abrt-addon
+Requires: container-exception-logger
```

Comment:
You can remove these
https://github.com/abrt/abrt/blob/bab1adb5426720e5c81a2ac2ccdf178cf907bf8c/abrt.spec.in#L1063
https://github.com/abrt/abrt/blob/bab1adb5426720e5c81a2ac2ccdf178cf907bf8c/abrt.spec.in#L1081